packages/cdc-react/src/components/Button/Button.tsx

```diff
@@ -96,15 +98,21 @@ export const Button = ({
   const disabled: boolean = state === "disabled" ? true : false;
 
   return (
-    <button
-      aria-label={ariaLabel}
-      id={id}
-      className={`cdc-react btn-wrap ${classes}`}
+    <div
```

Comment:
I want to get rid of this button wrap `div`.  I don't think it's necessary and goes against a11y guidelines.  What's the reason for re-introducing it?